portal-frontend/src/app/services/application-owner/application-owner.service.ts

```diff
@@ -100,4 +106,48 @@ export class ApplicationOwnerService {
     }
     return 0;
   }
+
+  async uploadCorporateSummary(files: File[]) {
+    if (files.length > 0) {
+      const file: File = files[0];
+
+      if (file.size > environment.maxFileSize) {
```

Comment:
definitely outside of this MR. But I feel like we need some more generic document service that could hold this logic and accept DTO. I think we have similar code now in 3 locations

Reply:
Done!  Cleaned up to emit a single file for each one selected, and added a new upload to DRY this up